kernel/siw_cm.h

```diff
@@ -57,9 +57,10 @@ enum siw_cep_state {
 };
 
 struct siw_mpa_info {
-	struct mpa_rr	hdr;	/* peer mpa hdr in host byte order */
-	char		*pdata;
-	int		bytes_rcvd;
+	struct mpa_rr		hdr;	/* peer mpa hdr in host byte order */
+	struct mpa_v2_data	enhanced_conn_data;
```

Comment:
I could but the RFC refers to it as enhanced connection data. That's why I named it that way.